README.md

```diff
@@ -71,3 +71,29 @@ The project uses pytest to run tests. To run the tests, run the following from y
 ```bash
 $ poetry run pytest
 ```
+
+## Provisioning a VM from an Ansible Control Node
+
+1. SSH into the control node
+2. Copy the `ansible` directory to the control node
+3. Replace the IP address in the `inventory` file with the IP address(es) of the managed VM(s)
+4. Create a file `ansible-pw.txt` containing the vault password
+5. Run the following command in the `ansible` directory, to provision the VM:
+    ```bash
+    $ ansible-playbook playbook.yml -i inventory --vault-password-file ansible-pw.txt
```

Comment:
This works nicely for us, but obviously for a real project I'd be very wary of "create a plain text file that contains a password" since that largely invalidates the point of the vault file..

OK, that's not quite true, since we can still check the vault file into our source control this way, but it does mean it's equivalent to putting a plain text file on the control node that doesn't get checked in

And maybe it's actually less of a concern since anyone logged in as our ansible user (e.g. `ec2-user` for us) can password-free SSH into the managed node which has the `.env` file on it anyway..

But there's still a chance that they reach our control node with a different less-privileged user, but still manage to view this file.

So maybe having written this all out, I weaken my statement a reasonable amount to - worth noting that using a plain text password file here increases our attack surface area, but I can't see any fundamental flaws!

I'll leave the comment because it's potentially an interesting chain of thought, even though the conclusion is that there's probably nothing worth changing here!